cmake/std/trilinosprhelpers/test/test_TrilinosPRConfigurationBase.py

```diff
@@ -0,0 +1,712 @@
+    output = "--- subprocess.check_call({})".format(", ".join(params))         # pragma: no cover
+
+    print("MOCK> mock_packageEnables_check_call()")                            # pragma: no cover
+    for k in args[0]:                                                          # pragma: no cover
+        print("    - '{}'".format(k))                                          # pragma: no cover
+    for k,v in kwargs.items():                                                 # pragma: no cover
+        print("    - {}={}".format(k,v))                                       # pragma: no cover
+    print("")                                                                  # pragma: no cover
+    return str.encode(output)                                                  # pragma: no cover
+
+
+def mock_subprocess_raise_CalledProcessError(*args, **kwargs):
+    cmd = " ".join(args[0])                                                    # pragma: no cover
+    print("MOCK: Raising subprocess.CalledProcessError")                       # pragma: no cover
+    print("    : Command is `{}`".format(cmd))                                 # pragma: no cover
+    raise subprocess.CalledProcessError(1, cmd)                                # pragma: no cover
+
+
+def mock_modulehelper_module_ok(*args, **kwargs):
+    """
+    Mock replacement for a ModuleHelper.module() call that succeeds
+    """
+    cmd = ", ".join(["'{}'".format(x) for x in args])
+    print("MOCK> module({})    QAPLA'!".format(cmd))                           # pragma: no cover
+    return 0                                                                   # pragma: no cover
+
+
+def mock_modulehelper_module_fail(*args, **kwargs):
+    """
+    Mock replacement for a ModuleHelper.module() call that fails
+    """
+    cmd = ", ".join(["'{}'".format(x) for x in args])
+    print("MOCK> module({})  # FAILS".format(cmd))                             # pragma: no cover
+    return 1                                                                   # pragma: no cover
+
+
+def mock_early_return(*args, **kwargs):
+    """
+    Mock helper that prints out the args and quits with nonzero return value.
+    """
+    print("MOCK>", args[0])                                                    # pragma: no cover
+    return 1                                                                   # pragma: no cover
+
+
+def mock_se_apply_pass(*args, **kwargs):
+    """
+    Mock for SetEnvironment.apply() call that would pass (i.e., return 0)
+
+    Returns:
+        int 0 for PASS
+    """
+    params = ", ".join(["{}={}".format(k,v) for k,v in kwargs.items()])         # pragma: no cover
+    print("MOCK> SetEnvironment.apply({})".format(params))                      # pragma: no cover
+    return 0                                                                    # pragma: no cover
+
+
+def mock_se_apply_fail(*args, **kwargs):
+    """
+    Mock for SetEnvironment.apply() call that would pass (i.e., return 1)
```

Comment:
fail

Reply:
@jmgate this is not a useful comment, please elaborate.

Reply:
Sorry, switch the word `pass` to `fail` in the docstring.